package/lib-mlpref_loadgen-static/install.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/bash
+
+#
+# CK installation script
+#
+# See CK LICENSE for licensing details.
+# See CK COPYRIGHT for copyright details.
+#
+
+# PACKAGE_DIR
+# INSTALL_DIR
+
+cd ${CK_ENV_MLPERF_INFERENCE_LOADGEN}
+echo "Loadgen source directory: ${CK_ENV_MLPERF_INFERENCE_LOADGEN}"
+cmake .
+cmake --build .
+
+if [ ! -d "${INSTALL_DIR}/lib" ]; then
+  mkdir ${INSTALL_DIR}/lib
+fi
+
+cp -f libmlperf_loadgen.a ${INSTALL_DIR}/lib
```

Comment:
This should ideally be done via `-DCMAKE_INSTALL_PREFIX=` e.g. see https://github.com/ctuning/ck-armnn/blob/master/package/lib-armnn/scripts.linux/install.sh#L131

However, we may need to make some modifications to `CMakeLists.txt` for this.

Reply:
See how the `install` command is used e.g. here: https://github.com/ARM-software/armnn/blob/branches/armnn_19_05/CMakeLists.txt#L419

Reply:
You can initially make such changes via a patch (in `patch.linux`), and then we can propose them for inclusion to the MLPerf repo.